static/app/views/projectInstall/createProject.tsx

```diff
@@ -130,10 +133,70 @@ const keyToErrorText: Record<string, string> = {
   detail: t('Project details'),
 };
 
+async function rollbackRulesAndProject({
```

Comment:
The whole flow should be treated as atomic: if any part fails - project, rule, or notification - everything should roll back. Otherwise, users might think something was created when it wasn’t. Even if they fix just one part (like a rule) but don’t change the project name or platform, clicking again can create duplicate projects. We should only redirect after a full success.